src/shared/ui/modal-window/modalWindow.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import { modal, modalClose, open, focusTrap } from '@/shared/lib/modal'
+import './modal-window.scss'
+import '@/shared/styel/helpers.scss'
+</script>
+
+<template>
+  <div
+    class="dialog-bg"
+    :class="{ open: open }"
+    :aria-hidden="!open"
+    @click="modalClose()"
+    @keydown.esc="modalClose()"
```

Comment:
Мы не вызываем эти функции, а назначаем их обработчиками событий.

```suggestion
    @click="modalClose"
    @keydown.esc="modalClose"
```